scripts/components/tool/tool-content.component.js

```diff
@@ -36,5 +42,11 @@ export default class {
 
   toggleError(hasError) {
     this.sankeyError.classList.toggle('is-hidden', !hasError);
+
+    if (hasError) {
```

Comment:
I think we can add the event listener on constructor, doesn't make sense to toggle add/remove depending on `hasError`

Reply:
moved